src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/services/DraftAdjudicationService.kt

```diff
@@ -291,6 +306,8 @@ class DraftAdjudicationService(
         offence.addAll(toReportedOffence(draftAdjudication.offenceDetails))
       }
       it.statement = draftAdjudication.incidentStatement!!.statement!!
+      it.transition(ReportedAdjudicationStatus.AWAITING_REVIEW)
```

Comment:
On reporting an adjudication (That has already been reported) we transition to AWAITING_REVIEW. This can be either because it was sent back, so it'll be in RETURNED state, or it has not yet been seen by a review and so it will be in AWAITING_REVIEW